designdocs/passkey_p256.md

```diff
@@ -0,0 +1,552 @@
+
+- Simplified Transaction Diagram
+  ![Transaction Diagram](https://private-user-images.githubusercontent.com/9152501/337430375-d20596c6-4c84-4a52-beb5-0647848a870c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc3MDYzMzcsIm5iZiI6MTcxNzcwNjAzNywicGF0aCI6Ii85MTUyNTAxLzMzNzQzMDM3NS1kMjA1OTZjNi00Yzg0LTRhNTItYmViNS0wNjQ3ODQ4YTg3MGMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYwNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MDZUMjAzMzU3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9Yjk1YTU4OGZkM2ZhODRkYTFmYjkwZmNiY2JmM2U2MjU3OTRmODQ3ZDQ0MTYzMGFiNzQzY2EwNjJiMmE5ODIxNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.abnMqctYF0ScbIxdQhr32xsjfH-M7P05bmAnrFj-TRM)
+
+## 4.2. Data maps for Legal Teams
+
+### Web Wallet backend
+
+- PassKey Signature of Account public key (`passkey_sig_pk`)
+- PassKey signature of transaction (`passkey_tx_signature`)
+- Frequency Account key signature of Passkey public key (`account_sig_passkey_pk`)
+- `credentialPublicKey`: Passkey Public key (`passkey_pk`)
+- `UserName`: User's website-specific username (email, etc...)
+- `UserId`: User's website-specific unique ID
+- `CredentialId`: The credential's credential ID for the pass key
+- `Counter`: The number of times the authenticator reported it has been used.
+- `credentialDeviceType`: Whether this is a single-device or multi-device credential.
+- `credentialBackedUp`: Whether or not the multi-device credential has been backed up. Always false for single-device credentials.
+- Transaction call
+
+### Users Device
+
+- Keypair for PassKey
+- Keypair seed backup for Frequency Account
+- All data included in the [Web Wallet backend](#web-wallet-backend) section
+
+## 5. Specification
+
+### Terms
+
+These are the following technical terms and specifications for the implementation of Passkey support on Frequency chain.
+
+- `account_keypair` : A cryptographic key pair generated from a seed phrase
+- `account_pk` : Account Public Key
+- `passkey_pk` : Passkey Public Key
+- `passkey_sig_pk` : Signature generated by Passkey on  `account_pk`
```

Comment:
If these keys are signing each other, I think they need to be named in similar ways, e.g. `passkey_sig_account_pk` and `account_sig_passkey_pk`